pkg/virt-handler/virtwrap/manager.go

```diff
@@ -328,21 +327,14 @@ func VMNamespaceKeyFunc(vm *v1.VM) string {
 	return domName
 }
 
-func (l *LibvirtDomainManager) SyncVM(vm *v1.VM) (*api.DomainSpec, error) {
-	var wantedSpec api.DomainSpec
-	mappingErrs := model.Copy(&wantedSpec, vm.Spec.Domain)
-
-	if len(mappingErrs) > 0 {
-		return nil, errors.NewAggregate(mappingErrs)
-	}
-
+func (l *LibvirtDomainManager) SyncVM(vm *v1.VM, domDesign *designer.DomainDesign) (*api.DomainSpec, error) {
```

Comment:
It just happens with disks, that when we resolve a PVC to an iSCSI connection, we're able to represent the iSCSI connection info in our public schema, but that won't be the case in general. Especially with networking, the info in the public domain spec is going to have no relation to the libvirt network XML - the libvirt network XML will never be exposed in the public domain spec. It will be similar with other types of device too.